### PR TITLE
EDSC-1296 Add metric to capture searches which result in user directly entering another search

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,15 +11,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160920144648) do
+ActiveRecord::Schema.define(version: 20170316054224) do
```
Comment (macrouch) [inline db/schema.rb]: shouldn't there be a migration for this?

File: app/controllers/search_controller.rb [hunk 2]
```diff
@@ -5,6 +5,9 @@ class SearchController < ApplicationController
   respond_to :json
 
   def extract_filters
+    isImmediateReentered = (params.delete :rerun) == 'true'
```
Comment (macrouch) [inline app/controllers/search_controller.rb]: this variable should be named `is_immediate_reentered`, along with `previous_keyword` below

File: db/schema.rb (hunk 1 above)
Comment (mightynimble) [inline db/schema.rb]: It is the migration that John added the other day. I think he just forgot to update the schema.rb.

Comment (macrouch) [inline db/schema.rb]: A migration was added?

Comment (mightynimble) [inline db/schema.rb]: https://github.com/nasa/earthdata-search/pull/372

Comment (macrouch) [inline db/schema.rb]: Ah, yep. Merged before I saw it

**Pull request merged**